### PR TITLE
Add coverage for ZEO module convenience functions

### LINKED ISSUE #102: mistake in writing
`ZEO/src/ZEO/__init__.py`

line 43 `ra
`should be `raise
`

### PR DESCRIPTION
**Pull request by tseaver**
Closes #102 after provoking it with a test.

### CODE REVIEW

File: src/ZEO/tests/testZEO.py [hunk 1]
```diff
@@ -1636,7 +1724,9 @@ def pack(self, t=None, referencesf=None):
         ZEO.ClientStorage._check_blob_cache_size(self.blob_dir, 0)
 
 def test_suite():
-    suite = unittest.TestSuite()
+    suite = unittest.TestSuite((
```
Comment (jamadden) [inline src/ZEO/tests/testZEO.py]: Is `suite = unittest.defaltTestLoader.loadTestsFromName(__name__)` an option here to avoid repetition?

Comment (tseaver) [reply src/ZEO/tests/testZEO.py]: Heh, have look at that `test_suite` function -- I don't have enough in my gumption reservoir to do anything other than the most minimal change to get my tests to run.

**Comment by tseaver** [comment]
> There are test failures in two environments, but they seem unrelated and have been happening for awhile

They are tests for @jimfulton's mta_acceptor / msgpack stuff, which I don't grok at all.

File: src/ZEO/tests/testZEO.py (hunk 1 above)
Comment (jamadden) [reply src/ZEO/tests/testZEO.py]: Hehe, fair enough. I didn’t want to deal with it either, I was just trying to pass the buck 😄

**Pull request merged**